src/Dap.Protocol/DebugAdapterConverters/DapClientNotificationConverter.cs

```diff
@@ -33,6 +33,16 @@ public override void WriteJson(JsonWriter writer, OutgoingNotification value, Js
                 writer.WritePropertyName("body");
                 serializer.Serialize(writer, value.Params);
             }
+            if (value.TraceParent != null)
+            {
+                writer.WritePropertyName("traceparent");
+                writer.WriteValue(value.TraceParent);
+            }
+            if (value.TraceState != null)
```

Comment:
[The spec](https://www.w3.org/TR/trace-context/#tracestate-header-field-values) discourages sending the `tracestate` header with an empty value. Should this condition be changed to `!string.IsNullOrEmpty(value.TraceState)`?

Reply:
Also, this block should be *inside* the `if (value.TraceParent != null)` block to satisfy [the spec](https://www.w3.org/TR/trace-context/#no-traceparent-received) that says `tracestate` without `traceparent` is invalid.

Reply:
Thanks!